website/content/docs/internals/telemetry.mdx

```diff
@@ -491,10 +491,8 @@ Since Autopilot runs only on the active node, these metrics are only emitted by
 | `vault.raft.state.candidate`    | Increments whenever raft server starts an election                                                            | Elections | counter |
 | `vault.raft.state.leader`       | Increments whenever raft server becomes a leader                                                              | Leaders   | counter |
 
-**Why they're important**: Normally, your raft cluster should have a stable
-leader. If there are frequent elections or leadership changes, it would likely
-indicate network issues between the raft nodes, or that the raft servers
-themselves are unable to keep up with the load.
+**Why are they vital?**: Raft cluster should have a stable leader during normal operations. If frequent elections or leadership changes occur, network issues between the raft nodes may be present or the raft servers
+cannot keep up with the load.
```

Comment:
```suggestion
**Why are they vital?**: If frequent elections or leadership changes occur, it would likely indicate network issues between the raft nodes or the raft servers cannot keep up with the load.
```